contrib/templates/experimental/scala-job/template/databricks.yml.tmpl

```
@@ -0,0 +1,52 @@
+# This is a Databricks asset bundle definition for {{.project_name}}.
+# See https://docs.databricks.com/dev-tools/bundles/index.html for documentation.
+bundle:
+  name: {{.project_name}}
```

```suggestion
  name: {{.project_name}}
  uuid: {{bundle_uuid}}
```

All the newer templates include a uuid, which can help for diagnostic and metrics use cases.